eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -536,30 +539,24 @@ jobs:
         targetFolder=$(Build.ArtifactStagingDirectory)/BuildLogs/
         mkdir -p ${targetFolder}
 
-        # Download rsync if using mariner
-        if grep -qE "CBL-Mariner|Azure Linux" /etc/os-release; then
-          sudo tdnf install -y rsync
-          sudo tdnf clean all
-        fi
-
         cd "$(sourcesPath)"
 
-        find artifacts/log/ -exec rsync -R {} -t ${targetFolder} \;
+        find artifacts/log/ -exec cp --parents {} ${targetFolder} \;
```

Comment:
@ellahathaway why did you switch away from rsync? `cp --parents` doesn't exist on macOS and this command fails right now (we don't notice because `find` doesn't fail).

Reply:
We might now be able to just use the `CopyFiles` AzDO task: https://github.com/dotnet/sdk/blob/14bb58e830dacafb644e2224c07965effeba6696/eng/pipelines/templates/jobs/vmr-build.yml#L501

We don't glob from the src folder anymore.

Reply:
https://github.com/dotnet/sdk/pull/45478

Reply:
@ViktorHofer ah thanks, I missed your comment before opening my PR as GitHub somehow didn't refresh. I think using CopyFiles@2 might be even simpler indeed.

Reply:
Tracked the original change down to [dotnet/installer@`242b4e2` (#12141)](https://github.com/dotnet/installer/pull/12141/commits/242b4e2590759a56e708ccd7ad6b4df340c9ae98) which said this
> Smoke tests set the HOME dir to a testing-smoke subdir which cause permissions issue with CopyFiles@2 on some distros.
> Manually copy the specific logs directory to workaround this.

We can try whether that's still an issue.

Reply:
I switched it because `rsync` is not available on all platforms. I'll look at your PR now.